js/utils/validateMediumEditor.js

```diff
@@ -6,7 +6,7 @@ function checkVal($field) {
     $field.val('');
   }
 
-  //replace double quotes with single quotes to avoid invalid json
+  // replace double quotes with single quotes to avoid invalid json
   fVal = fVal.replace(/\\([\s\S])|(")/g, "'");
```

Comment:
I don't think this should be necessary since when we serialize the data, the serializer handles stuff like this (doing any necessary escaping), whether we are serializing to formData or JSON.

But, I don't know the specific case it was put in for, so I'm leaving it in for now.